### PR TITLE
Korean Translation Core~Cycle 2

### PR DESCRIPTION
**Pull request by scappy64**
Starting with core~cycle 2, gradually catching up with global players.

### CODE REVIEW

File: translations/ko/pack/core_encounter.json [hunk 1]
```diff
@@ -1,626 +1,627 @@
...
+		"text": "<b>공개될 때</b>: Rhino가 입은 피해 4점을 치료합니다. 이렇게 치료된 피해가 없다면, 이 카드는 쇄도를 얻습니다."
 	},
 	{
 		"code": "01105",
-		"flavor": "\"Bring it!\" -- Rhino",
-		"name": "\"I'm Tough\"",
-		"text": "<b>When Revealed</b>: Give Rhino a tough status card. If Rhino already has a tough status card, this card gains surge."
+		"flavor": "\"덤벼라!\" -- Rhino",
+		"name": "\"난 튼튼하다!\"",
+		"text": "<b>공개될 때</b>: Rhino에게 강인함 상태 카드를 줍니다. Rhino에게 이미 강인함 상태 카드가 있다면, 이 카드는 쇄도를 얻습니다.
```
Comment (elkeinkrad) [inline translations/ko/pack/core_encounter.json:79]: json format issue
```suggestion
		"text": "<b>공개될 때</b>: Rhino에게 강인함 상태 카드를 줍니다. Rhino에게 이미 강인함 상태 카드가 있다면, 이 카드는 쇄도를 얻습니다."
```

File: translations/ko/pack/core_encounter.json [hunk 2]
```diff
@@ -1,626 +1,627 @@
...
-		"back_flavor": "Klaw has found a buyer for his illegal weapons. It's up to you to stop the sale but the meeting is surrounded by Klaw's cronies.",
-		"back_text": "<b>When Revealed</b>: Discard cards from the encounter deck until a minion is discarded. Put that minion into play engaged with the first player. Advance to stage 2B",
+		"back_flavor": "Klaw는 그의 불법 무기들을 구매할 고객을 찾았습니다. 거래를 막아야 하지만 회동 장소는 Klaw의 공범들이 철통같이 경계하고 있습니다.",
+		"back_text": "<b>공개될 때</b>: 조우 덱에서 하수인이 버려질 때까지 카드를 버립니다. 이렇게 버려진 하수인을 시작 플레이어와 교전 중인 채로 플레이 영역에 내려놓습니다. 스테이지 2B로 넘어갑니다",
 		"code": "01117",
-		"flavor": "Klaw is meeting with the Crimson Cowl. Klaw and the mysterious figure dart into the shadows when you confront them, and Klaw's minions move to cover their escape.",
-		"name": "Secret Rendezvous",
-		"text": "<b>If this stage is completed, the players lose the game.</b>"
+		"flavor": "Klaw는 Crimson Cowl과 만났습니다. 당신이 그들을 쫓자 Klaw와 신원 미상의 고객은 어둠 속으로 사라졌습니다. 그리고 Klaw의 하수인들이 그들의 탈출을 돕기 위해 움직입니다.",
+		"name": "비밀스러운 회동,
```
Comment (elkeinkrad) [inline translations/ko/pack/core_encounter.json:156]: json format issue

```suggestion
		"name": "비밀스러운 회동",
```

File: translations/ko/pack/core_encounter.json [hunk 3]
```diff
@@ -1,626 +1,627 @@
...
+		"text": "<b>이 스테이지가 달성되면, 플레이어들은 게임에서 패배합니다.</b>"
 	},
 	{
-		"attack_text": "<b>Forced Response</b>: After Klaw attacks and damages a character, stun that character.",
+		"attack_text": "<b>강제 반응</b>: Klaw가 공격하여 캐릭터에게 피해를 입힌 후, 그 캐릭터를 기절시킵니다.
```
Comment (elkeinkrad) [inline translations/ko/pack/core_encounter.json:160]: json format issue
```suggestion
		"attack_text": "<b>강제 반응</b>: Klaw가 공격하여 캐릭터에게 피해를 입힌 후, 그 캐릭터를 기절시킵니다.",
```

File: translations/ko/pack/core_encounter.json [hunk 4]
```diff
@@ -1,626 +1,627 @@
...
 		"code": "01118",
-		"name": "Sonic Converter",
-		"text": "Attach to Klaw.\n<b>Hero Action</b>: Spend [energy] [mental] [physical] resources → discard this card.",
-		"traits": "Weapon."
+		"name": "음파 변환기,
```
Comment (elkeinkrad) [inline translations/ko/pack/core_encounter.json:162]: json format issue
```suggestion
		"name": "음파 변환기",
```

**Review by elkeinkrad — CHANGES_REQUESTED** [review]
Here are some json format error.

**Comment by elkeinkrad** [comment]
Note to maintainer: This PR is written as unicode instead of ASCII. I think this is ok. However, if any problem occurs (unlike arkhamDB), the reason may be encoding issue.

File: translations/ko/pack/gob_encounter.json [hunk 5]
```diff
@@ -1,336 +1,337 @@
...
-		"text": "Attach to Green Goblin.\n<b>Hero Action</b>: Spend [mental] [mental] resources → discard this card.",
-		"traits": "Condition."
+		"name": "히스테리",
+		"scheme_text": "<b>강제 난입</b>: Green Goblin이 암약하거나 공격할 때, 이번 활성화에서 Green Goblin에게 강화 카드 1장을 추가로 줍니다.",
+		"text": "Green goblin에게 장착합니다.\n<b>히어로 행동</b>: [mental] [mental] 자원을 소모 → 이 카드를 버림.",
+		"traits": "상태."
 	},
 	{
-		"attack_text": "<b>Forced Response</b>: After the villain attacks you, discard Pumpkin Bombs and take 2 indirect damage.",
+		"attack_text": "<b>강제 반응</b>: 빌런이 당신을 공격한 후, 호박 폭탄을 버리고 간접 피해 2점을 입습니다."
```
Comment (elkeinkrad) [inline translations/ko/pack/gob_encounter.json:163]:
```suggestion
		"attack_text": "<b>강제 반응</b>: 빌런이 당신을 공격한 후, 호박 폭탄을 버리고 간접 피해 2점을 입습니다.",
```

**Review by elkeinkrad — CHANGES_REQUESTED** [review]

**Comment by Kamalisk** [comment]
unicode should be fine. looks like there is some invalid json somewhere though.

**Comment by scappy64** [comment]
> unicode should be fine. looks like there is some invalid json somewhere though.

Fixed!! Thank you.

**Comment by scappy64** [comment]
Due to Korean Copyright, I deleted all encounter card translation.

**Pull request merged**